portfolio/src/main/webapp/script.js

```diff
@@ -59,6 +59,31 @@ function displayCountry() {
     responsePromise.then(handleResponse);
 }
 
+function getComments() {
+    fetch('/data').then(response => response.json()).then((comments) => {
+        // comments is an object, not a string, so we have to
+        // reference its fields to create HTML content
+
+        console.log(comments);
+        const title = document.getElementById('content-title');
+        const content = document.getElementById('content-panel');
+
+        //Is setting this to empty a good practice standard?
```

Comment:
I'm not sure. Most developers quickly switch to a higher-level framework to handle the html element manipulation.

I suspect this will remove any child elements as a 'side effect' of setting it to the empty string, but it should be fine to skip this.